### PR TITLE
AC-405:Appointment Scheduling

### PR DESCRIPTION
**Pull request by paras7735**
JIRA issue Link https://issues.openmrs.org/projects/AC/issues/AC-405

### CODE REVIEW

**Comment by DefCon-007** [comment]
@paras7735 Thanks for the PR ! You did great work. Next time keep in mind to not use different-different PRs. :)

**Comment by paras7735** [comment]
I will fix the issue asap.

**Comment by paras7735** [comment]
@Sub6Resources I have fixed the issues which you were talking about
Please have a look.

File: openmrs-client/src/main/java/org/openmrs/mobile/models/appointment/Link________.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+
+package org.openmrs.mobile.models.appointment;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class Link________ {
```
Comment (dkayiwa) [inline openmrs-client/src/main/java/org/openmrs/mobile/models/appointment/Link________.java]: Did you intentionally come up with such class names?

**Comment by dkayiwa** [comment]
@paras7735 ping me to reopen when you resume working on this.

**Pull request closed without merging**